src/OrchardCore/OrchardCore/Shell/Distributed/DistributedShellHostedService.cs

```diff
@@ -153,42 +158,47 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                         continue;
                     }
 
-                    // Try to retrieve the tenant created global identifier from the distributed cache.
-                    string shellCreatedId;
+                    // Try to retrieve the tenant list changed global identifier from the distributed cache.
```

Comment:
I do not understand this comment.

Reply:
In a distributed environment, we have shared identifiers, one of them is changed each time the list of tenants changes, for example when a tenant is created and now when a tenant is removed.

Here Global meaning that it is not specific to a tenant but related to the list of all tenants.

Reply:
Not saying that my comment is good, my english is bad ;)

But there is only one global Identifier related to the list of all tenants.

This is the global identifier used to see if the tenant list changed.